src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -2067,13 +2073,22 @@ def _to_engine_output(output: str, return_code: int,
       output=bytes(truncated_output, 'utf-8'),
       return_code=return_code,
       timestamp=proto_timestamp)
+
+  if crash_path is None:
+    return engine_output
+  if os.path.getsize(crash_path) > 10 * 1024**2:
+    return engine_output
```

Comment:
should we log an error ? this seems pretty bad if we're not uploading a testcase right?

Reply:
It's just a testcase in logs. Really not that important.